dev/requirements.txt

```diff
@@ -2,5 +2,4 @@ flake8==3.5.0
 jira==1.0.3
 PyGithub==1.26.0
 Unidecode==0.04.19
-pypandoc==1.3.3
```

Comment:
I'm not sure what this is used for. @HyukjinKwon?

Reply:
I think this was added to install `pandoc` for R documentation as of https://github.com/apache/spark/pull/15309. So I think it's fine to keep it.

Reply:
I don't think it's needed for R. R needs pandoc. pypandoc is a Python wrapper around pandoc, and in most cases you need to separately install pandoc at the system level before you can install pypandoc.

I'm trying to confirm this myself, but the [two `Rscript` lines in the docs README](https://github.com/apache/spark/blob/b1f81f0072b06865ea1fdfe4c510474e64d39cfe/docs/README.md#prerequisites) take so long to run! I'll post an update when I've completed my test.

Reply:
OK, I've successfully managed to build, serve, and view the R API docs with pandoc installed on my system but _without pypandoc_, so I don't think we need this dependency.

cc @shaneknapp in case I am missing something from his perspective.

Reply:
@nchammas, yeah, I recall I am actually manually downloading it in the CI by using this option https://github.com/bebraw/pypandoc/tree/v1.3.3#installing-pandoc in the project I am working on.

Why don't you revive this line in this PR and investigate separately :)? This PR targets to remove pypandoc in PySpark.

What you're doing seems to remove or fix pandoc in SparkR.

Reply:
Well, I did retitle the Jira ticket and PR to say "eliminate pypandoc", so that's what I'm doing. 😄

My original purpose was just to update `setup.py`, but you nudged me to update a few more places unrelated to that, which expanded the scope a bit. So I did that, but I've expanded the scope one bit further to eliminate pypandoc entirely.

And this line is not specific to SparkR -- it's a generic list of dev requirements -- so I think it's on topic for the PR and Jira as they stand now.

Is there any further investigation you'd like me to do before you'd be comfortable accepting this change? I've already confirmed that I can build and view the R API docs.

Reply:
My concern is that https://github.com/apache/spark/pull/15309 seems working in Mac or Windows (assuming the pypandoc's documentation is correct).

Now we should add a bit of docs to at least explain that pandoc should be installed for R documentation generation because `rmarkdown` seems not bundling pandoc.